src/main/resources/messages_en.properties

```diff
@@ -440,3 +444,13 @@ transaction.update.invalid.notes.length=The max length of Notes is 255 character
 lot.deposit.input.null=Request object can not be null
 lot.deposit.instruction.invalid=Deposit Instructions are null or empty. Please check
 lot.input.instructions.for.non.present.units=The following unit(s): {0} are not assigned to the selected lots. 
+#Planting
+planting.request.null=Request object can not be null
+planting.preparation.empty=The provided filter does not return any entry to plant. Please check
+planting.lot.per.entry.no.null.or.empty=Lot per entry number can not be null or empty
+planting.withdrawals.per.unit.null.or.empty=Withdrawals per unit can not be null or empty
+planting.invalid.withdrawal.instruction=Invalid withdrawal instructions for Unit {0}. Withdraw all can be true only if group option is true. Amount cannot be specified if withdraw all is true. In any case, amount must be higher than zero
+planting.repeated.entry.no=Some entry numbers are repeated in the lotPerEntryNo list. Make sure that only one instruction per entryNo is provided
+planting.entry.no.invalid=Some entry numbers in lotPerEntryNo does not match the results of the selected observation units
+planting.lot.entry.no.invalid=Invalid lot {0} for the entryNo {1}. Lot may not exist, does not have same gid than the entry or be closed
```

Comment:
Maybe _"Lot may not exist, ~does not have~ **not have the** same gid than the entry or be closed"_